ecosystem/sep-0031.md

```diff
@@ -367,50 +343,15 @@ the corresponding off-chain asset, after fees have been applied.
 
 #### Asset Object Schema
 
-| Name                  | Type    | Description                                                                                                                                                                                                                                             |
-| --------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `sep12`               | object  | An object containing `sender` and `receiver` keys.                                                                                                                                                                                                      |
-| `min_amount`          | number  | (optional) Minimum amount. No limit if not specified.                                                                                                                                                                                                   |
-| `max_amount`          | number  | (optional) Maximum amount. No limit if not specified.                                                                                                                                                                                                   |
-| `fee_fixed`           | number  | (optional) A fixed fee in units of the Stellar asset. Leave blank if there is no fee or fee calculation cannot be modeled using a fixed and percentage fee.                                                                                             |
-| `fee_percent`         | number  | (optional) A percentage fee in percentage points. Leave blank if there is no fee or fee calculation cannot be modeled using a fixed and percentage fee.                                                                                                 |
-| `sender_sep12_type`   | string  | (**deprecated**, optional) The value of the `type` parameter the Sending Anchor should use for a `SEP-12 GET /customer` request. This field can be omitted if no KYC is necessary. Use a value from `sep12.sender.types` instead if any are present.    |
-| `receiver_sep12_type` | string  | (**deprecated**, optional) The value of the `type` parameter the Sending Anchor should use for a `SEP-12 GET /customer` request. This field can be omitted if no KYC is necessary. Use a values from `sep12.receiver.types` instead if any are present. |
-| `fields`              | object  | (**deprecated**, optional) An object containing the per-transaction parameters required in `POST /transactions` requests. Pass [SEP-9] fields via [SEP-12 PUT /customer](sep-0012.md#customer-put) instead.                                             |
```

Comment:
We need to keep deprecated fields in the document

Reply:
Hmm I thought these fields has been marked as deprecated for a long time. Is there's a plan/timeline to finally get rid of it?

Reply:
cc @JakeUrban I don't think we have a field removal strategy ATM

Reply:
Agreed with Gleb, we need to keep the deprecated fields in the doc for now.